pkg/openwsn/Makefile

```diff
@@ -5,6 +5,5 @@ PKG_VERSION=ff25e5d0ae5d344ed793a724d60532fb917bf1f8
 .PHONY: all
 
 all: git-download
-	"$(MAKE)" -C $(PKG_DIR)
```

Comment:
Hm, but that line should actually build the package. It is wrong and should be ```$(MAKE)" -C  $(PKG_BUILDDIR)```.

If you remove it, does the openwsn application build at all?